libs/ui/src/screens/token/token.tsx

```diff
@@ -52,14 +56,26 @@ export const Token: FC<Props> = ({ style }) => {
     params: { token }
   } = useRoute<RouteProp<ScreensParamList, ScreensEnum.Token>>();
   const { chainId } = useSelectedNetworkSelector();
+  const dispatch = useDispatch();
 
   const { name, symbol, tokenAddress, decimals, tokenId, thumbnailUri, balance } = token;
+
   const { price, usdPriceChange24h } = useTokenMarketInfoSelector(tokenAddress, chainId);
+  const { tokenBalance, fiatBalance } = useTokenBalance(tokenAddress, tokenId);
```

Comment:
why do you need `fiatBalance` from this hook if you are sending `token: TokenWithFiatBalance` as a navigation param and it has `fiatBalance` property?